eng/templates/Azure.Management.Template/content/test-resources.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0"
```

Comment:
This is not enough in me experimentation, resources node is required to be present.

Reply:
Added.